Entity/SnapshotManager.php

```diff
@@ -107,7 +107,9 @@ public function findEnableSnapshot(array $criteria)
 
         $query = $this->getRepository()
             ->createQueryBuilder('s')
-            ->andWhere('s.publicationDateStart <= :publicationDateStart AND ( s.publicationDateEnd IS NULL OR s.publicationDateEnd >= :publicationDateEnd )');
+            ->andWhere('s.publicationDateStart <= :publicationDateStart AND ( s.publicationDateEnd IS NULL OR s.publicationDateEnd >= :publicationDateEnd )')
+            ->andWhere('s.enabled = 1')
```

Comment:
IMHO `true` gives you a better understaning when reading the query, because `enabled` is a boolean field and no integer.